features/cli.feature

```diff
@@ -3,6 +3,21 @@ Feature: Command line interface
   As a person who wants to run features
   I want to run Cucumber on the command line
 
+  Scenario: run with no features directory
+    When I run cucumber.js with `-f summary`
+    Then the error output contains the text:
+    """
+    No such file or directory - features. You can use `cucumber --init` to get started.
+    """
+
+  Scenario: run a nonexistant feature file
+    Given a directory named "features" 
+    When I run cucumber.js with `-f summary features/a.feature`
+    Then the error output contains the text:
+    """
+    No such file or directory - features/a.feature. You can use `cucumber --init` to get started.
+    """
```

Comment:
I'd prefer we update this test to not suggest `cucumber --init`. For a new user, they may run `cucumber-js` before adding a features folder where we will suggest it, but once they get around to supplying a file, they should already have the structure in place.

Thus I'd prefer the logic: if no feature paths given and we hit an error, then suggest `cucumber --init`, otherwise don't.